diff --git a/node_modules/apexcharts/.npmignore b/node_modules/apexcharts/.npmignore
index 2c2dc49..3157d29 100644
--- a/node_modules/apexcharts/.npmignore
+++ b/node_modules/apexcharts/.npmignore
@@ -1,4 +1,11 @@
 node_modules/
 .DS_Store
 samples/
-issues/
\ No newline at end of file
+issues/2e
+tests/e2e/diffs
+.DS_Store
+.idea
+.nyc_output
+.vscode
+bundle-analysis.html
+package-lock.json
\ No newline at end of file
diff --git a/node_modules/apexcharts/src/charts/Line.js b/node_modules/apexcharts/src/charts/Line.js
index 3a7d382..4eff109 100644
--- a/node_modules/apexcharts/src/charts/Line.js
+++ b/node_modules/apexcharts/src/charts/Line.js
@@ -327,8 +327,8 @@ class Line {
           ...defaultRenderedPathOptions,
           pathFrom: paths.pathFromLine,
           pathTo: paths.linePaths[p],
-          stroke: lineFill,
-          // stroke: 'black',
+          // stroke: lineFill,
+          stroke: 'black',
           strokeWidth: this.strokeWidth,
           strokeLineCap: w.config.stroke.lineCap,
           fill: 'none'
